src/core/request.ts

```diff
@@ -119,6 +123,12 @@ export class Request {
       if (json.message.toLowerCase() === 'not authorized to view user') {
         return new IgPrivateUserError(response);
       }
+      if (json.message.toLowerCase().includes('too many requests')) {
```

Comment:
Is this a working case for languages other than default one (en_US)?

Reply:
https://github.com/dilame/instagram-private-api/blob/master/src/core/state.ts#L37

Reply:
Check other language. Unfortunately, works only for **en_US**.